src/ugenem/src/SendReportDialog.cpp

```diff
@@ -129,19 +129,11 @@ void ReportSender::parse(const QString& htmlReport, const QString& dumpUrl) {
         }
     }
 
-    QFile fp("/tmp/UGENEstacktrace.txt");
+    QFile fp(QDir::tempPath() + "/ugene_crashes/UGENEstacktrace.txt");
```

Comment:
Not related to the bug itself, but I found this problem too. This is the stacktrace, which was used before Google breakpad (it wasn't removed "just in case" and sometimes it really could show some extra info). Previously UGENE put it to `/tmp`, but this behavior was changed (see [here](https://github.com/ugeneunipro/ugene/blob/5f3edceb3e0c3dfe74d70b8a74d57f522508efce/src/corelibs/U2Private/src/crash_handler/CrashHandlerPrivateMac.cpp#L61)).